mne/io/nirx/nirx.py

```diff
@@ -223,11 +229,12 @@ def prepend(list, str):
             # Store channel location
             # Channel locations for short channels are bodged,
             # for short channels use the source location and add small offset
-            if (has_short > 0) & (len(np.where(short_det == det + 1)[0]) > 0):
-                info['chs'][ch_idx2 * 2]['loc'][:3] = src_locs[src, :]
-                info['chs'][ch_idx2 * 2 + 1]['loc'][:3] = src_locs[src, :]
-                info['chs'][ch_idx2 * 2]['loc'][0] += 0.8
-                info['chs'][ch_idx2 * 2 + 1]['loc'][0] += 0.8
+            if has_short > 0:
+                if len(np.where(short_det == det + 1)[0]) > 0:
```

Comment:
This is the same as `if det + 1 in short_det` right? If so it's clearer that it's just checking for membership.

Is this doing the right thing when `has_short > 0` and `det + 1 not in short_det` (i.e., don't adjust anything, as the `else` clause below isn't triggered either)?